src/app/cli/src/coda.ml

```diff
@@ -596,11 +596,12 @@ let daemon logger =
        in
        coda_ref := Some coda ;
        let%bind () = maybe_sleep 3. in
-       Coda_lib.start coda ;
        let web_service = Web_pipe.get_service () in
        Web_pipe.run_service coda web_service ~conf_dir ~logger ;
        Coda_run.setup_local_server ?client_whitelist ~rest_server_port
          ~insecure_rest_server coda ;
+       Coda_lib.start_proposer coda ;
+       Coda_lib.start_snark_worker coda ;
```

Comment:
It would be nice to keep `Coda_lib.start` one function, can we just decompose it in the right order inside Coda_lib?

Reply:
For the reason that it would be unfortunate to forget the call to one of these by accident.

Reply:
Okay will do that